app/domain/authentication/authn_jwt/signing_key/public_signing_keys.rb

```diff
@@ -0,0 +1,42 @@
+
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # This class is a POJO class presents public-keys structure
```

Comment:
What is POJO?

Reply:
[Plain old Java object](https://en.wikipedia.org/wiki/Plain_old_Java_object)